frontend/src/metabase/lib/dashboard_grid.js

```diff
@@ -6,7 +6,7 @@ export const GRID_MARGIN = 6;
 
 export const DEFAULT_CARD_SIZE = { width: 4, height: 4 };
 
-export const MIN_ROW_HEIGHT = 60;
+export const MIN_ROW_HEIGHT = 54;
```

Comment:
The difference is not so significant but why? It was mostly for text cards, especially with H1 text inside

Reply:
We used to calculate grid card dimensions in our own grid component. After we moved to `react-grid-layout` we don't have direct control over how dimensions are calculated. The grid receives `layout`, `width`, `rowHeight`, and `margin` and calculates the dimensions on its own. Looks like it has a bit different formula and RGL's cards had a slightly bigger height. I was able to tweak it only by decreasing the min row height here

Reply:
Thanks for the explanation!